File: Beomsu/Baekjoon_10250.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int t = scanner.nextInt();
        int[] results = new int[t];

        for (int i = 0; i < t; i++) {
            int H = scanner.nextInt();
            int W = scanner.nextInt();
            int N = scanner.nextInt();

            int floor = N % H == 0 ? H : N % H;
            int side = N % H == 0 ? N / H : N / H + 1;

            String sideText = side < 10 ? "0" + side : Integer.toString(side);

            results[i] = Integer.parseInt(floor + sideText);
        }

        for (int result : results) {
            System.out.println(result);
        }

        scanner.close();
    }
}

File: Beomsu/Baekjoon_10250.py
import sys

t = int(sys.stdin.readline())
result = []
for i in range(t):
    H,W,N = map(int,sys.stdin.readline().split())
    floor = H if N%H ==0 else N%H
    side = N//H  if N%H==0 else N//H +1

    sideText = "0" + str(side) if side < 10 else str(side)

    result.append(f"{floor}{sideText}")
for r in result:
    print(r)